lib/reek/tree_walker.rb

```diff
@@ -211,6 +218,14 @@ def visibility_modifier?(call_node)
       VISIBILITY_MODIFIERS.include?(call_node.method_name)
     end
 
+    # FIXME: Move to SendNode?
```

Comment:
Let's remove this comment and rather discuss it via github issue. Comments like that tend to get stale and confusing pretty quick....